File: robogit/src/main/java/org/robogit/config/DatabaseConfig.java
package org.robogit.config;

import org.springframework.context.annotation.Configuration;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.transaction.annotation.EnableTransactionManagement;

@EnableJpaRepositories(basePackages = "org.robogit.repository")
@Configuration
@EnableTransactionManagement
public class DatabaseConfig {
}

File: robogit/src/main/kotlin/org/robogit/config/DatabaseConfig.kt
package org.robogit.config

import org.springframework.context.annotation.Configuration
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.transaction.annotation.EnableTransactionManagement

@EnableJpaRepositories(basePackages = ["org.robogit.repository"])
@Configuration
@EnableTransactionManagement
open class DatabaseConfig {
  companion object {
    const val SCHEMA_NAME = "public"
  }
}
